cluster/examples/kubernetes/ceph/rook-operator-config.yaml

```diff
@@ -0,0 +1,22 @@
+kind: ConfigMap
```

Comment:
How about if we include the configmap directly in `operator.yaml` and `operator-openshift.yaml`? Then we can have example settings for those two environments available.

Reply:
We could also move all the example env vars into the configmap, but let's do that with a separate PR along with documentation for the settings in the configmap.